endpoints/currency_rates.go

```diff
@@ -0,0 +1,74 @@
+package endpoints
+
+import (
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/prebid/prebid-server/currencies"
+)
+
+// currencyRatesInfo holds currency rates information.
+type currencyRatesInfo struct {
+	Active           bool                           `json:"active"`
+	Source           *string                        `json:"source,omitempty"`
+	FetchingInterval *time.Duration                 `json:"fetchingIntervalNs,omitempty"`
+	LastUpdated      *time.Time                     `json:"lastUpdated,omitempty"`
+	Rates            *map[string]map[string]float64 `json:"rates,omitempty"`
+	AdditionalInfo   interface{}                    `json:"additionalInfo,omitempty"`
+}
+
+type rateConverter interface {
+	GetInfo() currencies.ConverterInfo
+}
+
+// newCurrencyRatesInfo creates a new CurrencyRatesInfo instance.
+func newCurrencyRatesInfo(rateConverter rateConverter) currencyRatesInfo {
+
+	currencyRatesInfo := currencyRatesInfo{
+		Active: false,
+	}
+
+	if rateConverter == nil {
+		return currencyRatesInfo
+	}
+
+	currencyRatesInfo.Active = true
+
+	infos := rateConverter.GetInfo()
+	if infos == nil {
+		return currencyRatesInfo
```

Comment:
Does it make sense for Active to be true but nothing else in the struct to be defined in this case?